rate/static/html/scale_item.html

```diff
@@ -0,0 +1,5 @@
+<label title="{level}">
```

Comment:
This was actually a fix from @cptvitamin. 

See: https://github.com/pmitros/RateXBlock/issues/4